src/olympia/devhub/templates/devhub/addons/submit/upload.html

```diff
@@ -31,6 +31,17 @@ <h3>{{ _('Upload Version') }}</h3>
     Use the fields below to upload your add-on package. After upload, a series
     of automated validation tests will be run on your file.
   {% endtrans %}
+  {% trans %}
```

Comment:
@diox will know better than me how this will actually get extracted but is this the "right" way to do this? I know there is some way of injecting a "variable" in a string but I don't remember exactly how to do it.

I think this could result in several independent strings for translators that could be a bit confusing..

Reply:
I think this might be an example. https://github.com/mozilla/addons-server/blob/8d00231f29890e4363f0b9e079823f4ff55a3429/src/olympia/amo/templates/amo/404.html#L7-L38

I guess you can pass arguments to the `{% trans foo="bar" %}` which allows you to encapsulate a single "string" with replaceable content. Then you can just reference the content.

Again @diox please correct me if I'm wrong.

Reply:
Correct, we don't want to break a translated string like this. It's not just about it not making sense for translators: some languages might need to change the order of the words, which can't happen if it's been broken down into separate strings.

Reply:
As a side note that 404 page is a bit special because the entire page contents is in the `{% trans %}` block, which is fairly rare. That pretty much only happens for error pages. A simpler example:
https://github.com/mozilla/addons-server/blob/b8ffdae3c8a459ef61525aefa21c4debbb45896b/src/olympia/devhub/templates/devhub/suppressed_email.html#L4-L6

Another which is old and kinda bad, but shows that it works even with a `|filter` applied to the variable assignment:
https://github.com/mozilla/addons-server/blob/b8ffdae3c8a459ef61525aefa21c4debbb45896b/src/olympia/devhub/templates/devhub/addons/dashboard.html#L42-L44